setup.py

```diff
@@ -9,6 +9,7 @@
     "chiapos==0.12.42b2",  # proof of space
     "clvm==0.7",
     "clvm_tools==0.2.0",
+    "clvm_rs_test>=0.1.0",
```

Comment:
I uploaded this package for MacOS as a test, until we can get one uploaded by our pypi account (which I believe requires chia's pypi token)